source/administration/changelog.md

```diff
@@ -228,10 +228,16 @@ mattermost-redux:
 - [crspeller](https://github.com/crspeller), [csduarte](https://github.com/csduarte)
 
 
-## Release v3.7.3
+## Release v3.7.5
 
 ### Notes on Patch Release
 
+ - **v3.7.5, released 2017-04-27**
+   - Fixed a number of security issues, and [upgrading](http://docs.mattermost.com/administration/upgrade.html) is highly recommended. Details will be posted on our [security updates page](https://about.mattermost.com/security-updates/) 14 days after release as per the [Mattermost Responsible Disclosure Policy](https://www.mattermost.org/responsible-disclosure-policy/)
+     - Note: The **System Console > Configuration > [Site URL](../../administration/config-settings.html#site-url)** field is now mandatory. Set the Site URL in the System Console, or in the `gitlab.rb` file if you are using GitLab Mattermost.
+     - Note: Upgrading will invalidate existing team invite links and email verifications.
```

Comment:
Yes I confirmed with CS. It was reduced from 200 to 64. But it didn't actually change from the users experience since the UI limited it to 64 characters anyway.